### DISCUSSION TITLE
I'm modifying the looks of my homepage. I'm logged into an admin account. I wish to change the background image. Should I change it on the AWS app or can I change it locally?

### DISCUSSION THREAD

**Discussion by animanoir** [mozilla/hubs#2788] [General Questions & Discussion] [answered]
Like, whatever change I deploy from my local server will be saved in the AWS app?

**Comment by Thitle** [comment, marked as answer]
If you have a HUB CLOUD on your home page on your website, you have the ADMIN icon, so you can make the change you need.
